### DISCUSSION TITLE
Is there a way to override default blade components like 'sidebar'

### DISCUSSION THREAD

**Discussion by nabileisa** [filamentphp/filament#1782] [Help] [answered]
Hi, thank you for the great work on this package, 
I have a question , Is there an easy way yo override existing blade components without modifying core package file,
I would like for example to have a dark sidebar with custom logo and few other customizations that won't be possible using tailwind theme only , is there a way to do this?
thanks.

**Comment by nabileisa** [comment, marked as answer]
Never mind , just figured it out by publishing the views using `php artisan vendor:publish --tag="filament-views"` and keeping the views that I needed to modify only. I couldn't figure it out at first since I couldn't find the command or the tag for the views in the docs.
Thanks again for the great package.

**Reply by danharrin** [reply]
Cool. Please be aware that future releases of Filament could break your overriden views. Try using the `filament-*` CSS classes only to make your customisations.

**Reply by nabileisa** [reply]
Will keep that in mind ,If possible will use the classes , if not then updating  overriden views will become part of the upgrade process , thanks for the the response
